### ISSUE TITLE
2.0 needs an example for instatiating the generator

### ISSUE THREAD

**Issue by tacman** [FakerPHP/Faker#508]
### Summary

Bumping the library to 2.x-dev fails

```
        $generator = Faker\Factory::create();
```

Can't get the provider Address in local en_US.  I imagine it's something that not installed or set up.  I tried installing fakerphp/core, but that didn't help.

### Versions

|                | Version |
|:---------------|:--------|
| PHP            | 8.1   |
| `fakerphp/faker`  | 2.x-dev   |

### Self-enclosed code snippet for reproduction

```php
# replace this with a self-enclosed usage example
        $generator = Faker\Factory::create();

```

### Expected output

```txt
no errors
```

### Actual output

```txt

```

**Comment by pimjansen** [comment]
Its also not ready so thats the main reason. What are you trying to do with the in-dev version?

**Comment by tacman** [comment]
I'm working on a Symfony bundle that exposes phpfaker methods to twig.

https://github.com/survos/SurvosFakerBundle

 I did a few dozen calls individually, then I decided to extract the
parameters and return types automatically.  Since version 1 is php 7.1, I
had to parse the docBlocks, but I saw the v2 was php 7.4 so was easier to
reflect.

Ultimately, I hacked a solution together to generate  a demo and the
documentation, which I'll post shortly.

PHP Faker is a great library, thanks for it!
